shared/git/row/index.js

```diff
@@ -185,6 +195,43 @@ class Row extends React.Component<Props, State> {
                 <Text type="BodySmall">.</Text>
               </Text>
             </Box>
+            {!!this.props.teamname && (
+              <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', marginTop: globalMargins.xtiny}}>
+                {this.props.canEdit && (
+                  <Checkbox
+                    checked={!this.props.chatDisabled}
+                    onCheck={this.props.onToggleChatEnabled}
+                    label=""
+                    labelComponent={
+                      <Text type="BodySmall">
+                        Announce pushes in{' '}
+                        <Text
+                          type={this.props.chatDisabled ? 'BodySmall' : 'BodySmallPrimaryLink'}
+                          onClick={e => {
```

Comment:
better to do this in the connector, you can use withHandlers tomake a thing that takes in an event so you can cancel it